shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -1332,7 +1332,7 @@ - (void)onPreferredStatusBarStyleUpdated:(NSNotification*)notification {
 
 #pragma mark - Platform views
 
-- (flutter::FlutterPlatformViewsController*)platformViewsController {
+- (std::shared_ptr<flutter::FlutterPlatformViewsController>)platformViewsController {
```

Comment:
Is this getting called at least once per frame?  If so it may be desirable to pass back a const reference to allow any copying of the shared_ptr.